internal/cli/performanceadvisor/namespaces/list.go

```diff
@@ -66,33 +62,15 @@ func (opts *ListOpts) newNamespaceOptions() *atlas.NamespaceOptions {
 	}
 }
 
-func (opts *ListOpts) validateProcessName() error {
-	const length = 2
-	process := strings.Split(opts.processName, ":")
-	if len(process) != length {
-		return fmt.Errorf("'%v' is not valid", opts.processName)
-	}
-	return nil
-}
-
 func (opts *ListOpts) host() (string, error) {
```

Comment:
[q] any reason why PerformanceAdvisorOpts can't implement this method?
If it can actually do it then you can make the validation function private